.pipelines/pipeline.user.windows.official.yml

```diff
@@ -16,13 +16,11 @@ build:
     - !!buildcommand
       name: 'Assemble ApplicationInsights-Java JARs'
       command: '.scripts/gradle.cmd'
-      arguments: 'assemble copyLibsToGlobalArtifactsFolder -DisRelease=true -Pai.etw.native.build=release'
+      arguments: 'publishToMavenLocal -DisRelease=true -Pai.etw.native.build=release'
```

Comment:
why do you want to use 'publishToMavenLocal' for builds? can we keep the old one? what is the reason of this change?

Reply:
publishToMavenLocal is the standard way to build javadoc/sources/pom, what we were doing previously was non-standard